e2e/MockStdinWatchPlugin.js

```diff
@@ -24,4 +24,13 @@ class MockStdinWatchPlugin {
     });
   }
 }
+
+/**
+ * Tried re-writing this in typescript but the tests just failed
```

Comment:
can you replace this comment with "Watch plugins are not transpiled" or something like it?